homeassistant/components/openerz/sensor.py

```diff
@@ -0,0 +1,70 @@
+"""Support for OpenERZ API for Zurich city waste disposal system."""
+from datetime import timedelta
+import logging
+
+from openerz_api.main import OpenERZConnector
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.config_validation import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(hours=12)
+
+CONF_ZIP = "zip"
+CONF_WASTE_TYPE = "waste_type"
+CONF_NAME = "name"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_ZIP): cv.positive_int,
+        vol.Required(CONF_WASTE_TYPE, default="waste"): cv.string,
+        vol.Optional(CONF_NAME): cv.string,
+    }
+)
+
+
+async def async_setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
Please make this a normal function, ie not a coroutine, and rename it to `setup_platform`. The library doesn't support asyncio.